protocol/communities/manager.go

```diff
@@ -360,7 +377,82 @@ func (m *Manager) runENSVerificationLoop() {
 					return
 				}
 				m.logger.Info("received records", zap.Any("records", records))
+			}
+		}
+	}()
+}
+
+func (m *Manager) runOwnerVerificationLoop() {
```

Comment:
could you please check `TestTokenMasterReceiveOwnerTokenFromControlNode` test?
I was unable to make it work after the control node deployed ownerToken and owner permission
It seems this loop is not starting
P.S. I did additional set up for test case:
```
s.collectiblesServiceMock.SetSignerPubkeyForCommunity(community.ID(), common.PubkeyToHex(&s.controlNode.identity.PublicKey))
communities.SetValidateInterval(50 * time.Millisecond)
```

Reply:
I need to fix the test, yes, it's because you don't get it through retrieveall, but it will be published eventually as a signal, so I think I need to adjust the test, I will look into it, well spotted, thank you